code/game/machinery/OpTable.dm

```diff
@@ -164,6 +165,17 @@
 	take_victim(usr,usr)
 
 /obj/machinery/optable/attackby(obj/item/weapon/W as obj, mob/living/carbon/user as mob)
+	if(iswrench(W))
+		playsound(get_turf(src), 'sound/items/Ratchet.ogg', 50, 1)
+		if(rating==1)
+			new /obj/item/weapon/stock_parts/scanning_module(src.loc)

```

Comment:
I'm still not happy with this method of building optables generally.
There's no real reason to snowflake it in.